File: LeetCode_2348/Main.py
class Solution(object):
    def zeroFilledSubarray(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        cnt = 0
        ans = 0
        for num in nums:
            if num == 0:
                cnt += 1
            else:
                ans += int(cnt * (cnt + 1) / 2)
                cnt = 0
        ans += int(cnt * (cnt + 1) / 2)
        return ans

File: LeetCode_2348/Main.java
package LeetCode_2348;

class Solution {
    public long zeroFilledSubarray(int[] nums) {
        long cnt = 0, ans = 0;
        for (int num : nums) {
            if (num == 0) {
                cnt++;
            } else {
                ans += cnt * (cnt + 1) / 2;
                cnt = 0;
            }
        }
        ans += cnt * (cnt + 1) / 2;
        return ans;
    }
}

public class Main {
    public static void main(String[] args) {
        
    }
}
